virttest/virt_vm.py

```diff
@@ -668,6 +671,17 @@ def get_address(self, index=0):
                 be for each wait_for_login() call.
                 """
                 devs = set([nic.netdst])
+            if not arp_ip:
+                if (len(macs) > 1) and flexible_index:
+                    # Try to poke for each index
+                    for idx in range(len(macs)):
+                        try:
+                            mac = self.get_virsh_mac_address(idx)
+                        except IndexError:
```

Comment:
@sathnaga 
for tp-qemu test VM not create with libvirt, I guess self.get_virsh_mac_address() will raise TestFail exception, it will block tp-qemu  test.